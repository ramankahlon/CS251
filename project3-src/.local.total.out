CS251: . kahlonr Sun Feb 11 17:32:03 EST 2018  Total:  6 of 10
CS251: . kahlonr Sun Feb 11 17:52:05 EST 2018  Total:  6 of 10
CS251: . kahlonr Sun Feb 11 18:05:25 EST 2018  Total:  6 of 10
