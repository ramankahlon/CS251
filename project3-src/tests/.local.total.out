CS251: . elkindi Sat Feb 3 14:34:17 EST 2018  Total:  140 of 140
CS251: . elkindi Sat Feb 3 14:37:44 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:37:50 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:39:37 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:41:21 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:42:17 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 14:42:25 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 14:58:29 EST 2018  Total:  0 of 88
CS251: . elkindi Sat Feb 3 14:59:29 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:59:34 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 14:59:50 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 15:00:57 EST 2018  Total:  88 of 88
CS251: . elkindi Sat Feb 3 15:02:53 EST 2018  Total:  112 of 112
CS251: . elkindi Sat Feb 3 15:05:14 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 15:05:33 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 15:12:14 EST 2018  Total:  2 of 10
CS251: . elkindi Sat Feb 3 15:15:46 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:16:11 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 15:17:42 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:18:05 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:20:07 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:25:12 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:31:16 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:32:21 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:13 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:59 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:43 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:48 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:45:15 EST 2018  Total:  0 of 80
CS251: . elkindi Sat Feb 3 15:48:47 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:49 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:52 EST 2018  Total:  0 of 80
CS251: . kahlonr Mon Feb 12 15:20:22 EST 2018  Total:  1 of 10
CS251: . kahlonr Mon Feb 12 16:16:51 EST 2018  Total:  1 of 10
CS251: . kahlonr Tue Feb 13 21:46:24 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 00:35:39 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 00:35:51 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 00:36:00 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 00:37:28 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 13:19:19 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 13:19:31 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 13:19:39 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 13:20:34 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 13:25:13 EST 2018  Total:  0 of 0
CS251: . kahlonr Tue Feb 27 13:25:22 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 13:25:33 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 13:25:43 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 13:53:57 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 14:14:47 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 20:07:50 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 20:08:01 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 20:08:09 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 20:13:39 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 20:13:49 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 20:13:57 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 21:06:03 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 21:06:12 EST 2018  Total:  10 of 10
CS251: . kahlonr Tue Feb 27 21:06:20 EST 2018  Total:  0 of 80
CS251: . kahlonr Tue Feb 27 23:41:32 EST 2018  Total:  76 of 80
CS251: . kahlonr Tue Feb 27 23:45:44 EST 2018  Total:  76 of 80
